system/Test/FilterTestTrait.php

```diff
@@ -19,6 +19,7 @@
 use CodeIgniter\Router\RouteCollection;
 use Config\Filters as FiltersConfig;
 use Config\Services;
+use Closure;
```

Comment:
sort imports

Reply:
I always put root classes last - is that not proper sorting?

Reply:
@paulbalandan See https://github.com/codeigniter4/coding-standard/pull/42

Reply:
We sort alphabetically.

Reply:
Gross. 😜 Will change. 

@kenjis thanks for working on the standard.